### PR TITLE
Issue 3198 - Remove out of sync version for Apicurito Fuse builder

### LINKED ISSUE #3198: Remove out of sync version number for Apicurito Fuse creator
### General information

see https://issues.jboss.org/browse/ENTESB-9926

  * Minishift version: 
  * OS: Linux / macOS / Windows
  * Hypervisor: KVM / xhyve / Hyper-V / VirtualBox

### Steps to reproduce

  1. 
  2. 
  3. 
  4. 

### Logs

You can start Minishift with `minishift start --show-libmachine-logs -v5` to collect logs.
Please consider posting this on http://gist.github.com/ and post the link in the issue.

### PR DESCRIPTION
**Pull request by apupier**
Signed-off-by: Aurélien Pupier <apupier@redhat.com>

Fixes #3198 

Steps to test the pull request

  1. 
  2. 
  3. 
  4.

### CODE REVIEW

**Comment by apupier** [comment]
Can someone review this PR please?

I rebased on master. The same issue has been fixed upstreams https://github.com/jboss-fuse/application-templates/pull/62 https://github.com/Apicurio/apicurito/pull/31

**Comment by apupier** [comment]
anyone to move this PR forward?

seems the "ci.centos.org PR build" is not automatically triggered.

I don't have rights to merge the PR even if it has been approved.

**Comment by tsedmik** [comment]
ok to test

**Comment by agajdosi** [comment]
ok to test

**Comment by apupier** [comment]
the failure seems unrelated to my PR modifications:

```
09:01:53 - build/troubleshooting/troubleshooting-driver-plugins.html
09:01:53   *  External link https://blogs.msdn.microsoft.com/virtual_pc_guy/2010/09/28/creating-a-hyper-v-administrators-local-group-through-powershell failed: 410 No error
09:01:53 - build/using/static-ip.html
09:01:53   *  External link https://blogs.technet.microsoft.com/virtualization/2016/05/25/windows-nat-winnat-capabilities-and-limitations failed: 410 No error
09:01:53 htmlproofer 3.7.2 | Error:  HTML-Proofer found 2 failures!
```

**Comment by tsedmik** [comment]
ok to test

**Comment by tsedmik** [comment]
the PR is blocked with #3220

**Comment by tsedmik** [comment]
ok to test

**Comment by apupier** [comment]
another failure, now due to https://github.com/minishift/minishift/issues/3227

File: addons/xpaas/v3.10/xpaas-templates/fuse-apicurito.yml [hunk 1]
```diff
@@ -64,7 +64,7 @@ objects:
       var ApicuritoConfig = {
           "generators": [
               {
-                  "name":"Fuse 7.1 Camel Project",
+                  "name":"Fuse Camel Project",
```
Comment (praveenkumar) [inline addons/xpaas/v3.10/xpaas-templates/fuse-apicurito.yml]: 👍

File: addons/xpaas/v3.11/xpaas-templates/fuse-apicurito.yml [hunk 2]
```diff
@@ -64,7 +64,7 @@ objects:
       var ApicuritoConfig = {
           "generators": [
               {
-                  "name":"Fuse 7.1 Camel Project",
+                  "name":"Fuse Camel Project",
```
Comment (praveenkumar) [inline addons/xpaas/v3.11/xpaas-templates/fuse-apicurito.yml]: 👍

**Pull request merged**